Sources/Extensions/UIViewController+Child.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  UIViewController+Child.swift
+//  Keychain
+//
+//  Created by Jelle Vandebeeck on 11/02/2018.
+//
+
+import UIKit
+
+extension UIViewController {
+    
+    /// Add a given controller to the current controller as a child view controller. The given
+    /// controller's view is added as a subview to the given `UIView`.
+    public func add(controller: UIViewController, to view: UIView) {
```

Comment:
🎈would call the argument `viewController` iso `controller`, but seems like 🍎 themselves aren't really consistent in that naming (eg `func addChildViewController(_ `**childController**`: UIViewController)` vs `func overrideTraitCollection(`**forChildViewController childViewController**`: UIViewController) -> UITraitCollection?`)